quality_partner/views/res_partner_views.xml

```diff
@@ -32,6 +32,10 @@
                             </group>
                             <group string="Documents">
                                 <field name="quality_document_ids" nolabel="1" />
+                                <field
+                                    name="quality_document_document_type_ids"
+                                    invisible="1"
+                                />
```

Comment:
@MiquelRForgeFlow this shouldn't be necessary because that field is not used directly by the view. I tested it and it's not needed indeed, the constraint is triggered anyway.

Reply:
ok, updated!